rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/adapter/CovidPatientHistoryAdapterTest.java

```diff
@@ -0,0 +1,83 @@
+package io.ona.rdt.adapter;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.ona.rdt.PowerMockTest;
+import io.ona.rdt.domain.PatientHistoryEntry;
+import io.ona.rdt.viewholder.CovidPatientHistoryViewHolder;
+
+public class CovidPatientHistoryAdapterTest extends PowerMockTest {
+
+    private final String patientKey = "key";
+    private final String patientVal = "val";
+
+    private CovidPatientHistoryAdapter adapter;
+    private List<PatientHistoryEntry> list;
+
+    @Before
+    public void setUp() {
+
+        list = new ArrayList<>();
+        list.add(new PatientHistoryEntry(patientKey, patientVal));
+
+        adapter = new CovidPatientHistoryAdapter(list);
+    }
+
+    @Test
+    @PrepareForTest(LayoutInflater.class)
+    public void testOnCreateViewHolder() {
+
+        PowerMockito.mockStatic(LayoutInflater.class);
```

Comment:
Any reason not to extend RobolectricTest which would remove the need to mock the LayoutInflater?

Reply:
fixed. now it's extended to the RobolectricTest